test/util/testnode/config.go

```diff
@@ -82,7 +82,17 @@ func (c *Config) WithSuppressLogs(sl bool) *Config {
 	return c
 }
 
-// WithTimeoutCommit sets the TimeoutCommit and returns the Config.
+// WithBlockTime sets the block time and returns the Config.
+func (c *Config) WithBlockTime(d time.Duration) *Config {
+	c.TmConfig.Consensus.TimeoutCommit = d
+	creator := DefaultAppCreator(d)
+	c.WithAppCreator(creator)
+	return c
+}
```

Comment:
added this to help fix the insanely flakey square size test

Reply:
Can we not still call this TimeoutCommit because it changes the TimeoutCommit not the BlockTime

Reply:
sorry, I'm not understanding this comment completely. the reasoning for this change was because we were overriding the timeout

Reply:
ahh I get it now thanks, good point

https://github.com/celestiaorg/celestia-app/pull/4008/commits/c53ee35822cafa8eef14a590198d2dc7a4f0d816